packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -151,7 +152,7 @@ export default class RtcMetrics {
    */
   private resetConnection() {
     this.connectionId = uuid.v4();
-    this.shouldSendMetricsOnNextStatsReport = true;
+    this.shouldSendMetricsOnNextStatsReport = 1;
```

Comment:
we should set this to 30000/5000 here too, right? This is called from constructor so this is the default used when we join the meeting straight in, without lobby.